apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data.Tests/Google.Cloud.Spanner.Data.Tests.xproj

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm hoping it'll be this quarter. I don't expect it to cause much disruption to you though... I'll probably do GAX first, then all of google-cloud-dotnet in one go... but upgrading the Spanner part should be simple enough either before or after we merge it into master.

(If you haven't already got VS2017 installed, plan to do that :)